modules/Kconfig.nuvoton

```diff
@@ -75,4 +75,8 @@ menu "Nuvoton NuMaker drivers"
 		bool "NuMaker RMC"
 		help
 		  Enable Nuvoton RMC HAL module driver
+	config HAS_NUMAKER_RTC
+		bool "NuMaker RTC"
+		help
+		  Enable Nuvoton RTC HAL module driver
```

Comment:
You may be interested in asking DTS whether you have your RTC enabled, for example the way we do that for Nordic:
https://github.com/zephyrproject-rtos/zephyr/blob/2765d23775146c58b91c5602664308f777fb0346/modules/hal_nordic/nrfx/Kconfig#L268-L274

Reply:
Thanks of your suggestion.
My opinion is: hal is to support zephyr driver. So, Kconfig.numaker of zephyr rtc driver will asking  DTS and select rtc's hal config.